### DISCUSSION TITLE
Allow IP CIDR ranges in role IP access

### DISCUSSION THREAD

**Discussion by Joey92** [directus/directus#13853] [Feature Requests]
I want to use the role's IP access feature to allow a bot to access the Directus API, but I can't really be sure under which IP the request will come in.

In my use case I would need to add a docker IP CIDR range so the role allows IPs in the 172.17. 0.0/16 range.

Looking at the code, we match the IP exactly: https://github.com/directus/directus/blob/main/api/src/middleware/check-ip.ts#L17

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
